src/services/impl/WindowsFileSystem.cpp

```diff
@@ -60,18 +77,26 @@ size_t WindowsFileSystem::GetFilesInDirectory(const std::wstring& sDirectory, _I
 
 bool WindowsFileSystem::DeleteFile(const std::wstring& sPath) const noexcept
 {
-    return static_cast<bool>(DeleteFileW(sPath.c_str()));
+    std::wstring sBuffer;
+    const auto& sAbsolutePath = MakeAbsolute(sBuffer, sPath);
+    return static_cast<bool>(DeleteFileW(sAbsolutePath.c_str()));
```

Comment:
I'm not totally sure if the type.1b PR will overwrite this but you should probably just use `!= 0` (0 is failure for these File functions) instead of casting.

If it can be done by merging, don't worry about it. There doesn't seem to be anything else that needs improving here.

Reply:
Since the same line is modified in two different ways, it'll cause a merge conflict, which will have to be resolved manually.